src/Simulation/QSharpFoundation/Bitwise/Bitwise.qs

```diff
@@ -75,15 +75,38 @@ namespace Microsoft.Quantum.Bitwise {
 
 
     /// # Summary
-    /// Returns the bitwise PARITY of an integer (1 if its binary representation contains odd number of ones and 0 otherwise).
+    /// Returns the bitwise PARITY of an integer
+    /// (1 if its [two's complement](https://en.wikipedia.org/wiki/Signed_number_representations#Two's_complement)
+    /// representation contains odd number of ones and 0 otherwise).
```

Comment:
Some clients (e.g.: Visual Studio) don't support rich formatting in summaries, such that for maximum portability, it's generally best to keep the summary to plain text only and to defer links and other formatting to a `/// # Description` section:
```suggestion
    /// Returns the bitwise parity of an integer.
    ///
    /// # Description
    /// This function outputs the bitwise parity of the 
    /// [two's complement](https://en.wikipedia.org/wiki/Signed_number_representations#Two's_complement)
    /// representation of its input, returning `1` if that representation
    /// contains an odd number of ones, and returning `0` otherwise.
```

Reply:
Fixed.